packages/cyberstorm/src/components/componentStyles/Breadcrumb.module.css

```diff
@@ -0,0 +1,24 @@
+.breadcrumb {
+  display: flex;
+  flex-direction: row;
+  gap: var(--size-s);
+  align-items: center;
+  font-size: var(--size-m);
+  font-family: Inter, sans-serif;
+}
+
+.breadcrumbLabel {
```

Comment:
IIRC it was also decided that we shouldn't use the repetitive prefixes, so this should be just `.label`.